plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -268,7 +268,15 @@ public void setImageName(final String imageName) {
 		final DockerImageURI uri = new DockerImageURI(imageName);
 		setResourceName(dockerImage2OpenshiftResourceConverter.convert(uri));
 	}
-	
+
+	@Override
+	public void setImageName(String imageName, boolean forceUpdate) {
+		if(forceUpdate && this.imageName != null && this.imageName.equals(imageName)) {
```

Comment:
@adietish , it seems I understood, why in this case null as old value does not work. 
BeanPropertyListener in propertyChange when computing 'diff'sets it to null, if oldValue is null.
SimplePropertyObservableValue, when 'diff' is null uses cachedValue as old value, and cachedValue is the value taken from model which equals to the new fired value in our specific case. As the result the event is not send to the Text.